### PR TITLE
Add bioconductor banocc recipe

### PR DESCRIPTION
**Pull request by ljmciver**
Hello bioconda team members,

Thank you for the great new resource that is bioconda. The documentation for contributing a new recipe was fantastic. The bioconductor skeleton script was very useful.

This PR adds a new recipe for BAnOCC. It is a R package relevant to the biological sciences. For more information see the BAnOCC documenation,  http://huttenhower.sph.harvard.edu/banocc . 

I have tested the package with the simulate travis script locally and it passed without errors. I tried the travis ci tests but ended up having issues linting other packages. 

Please let me know if you have any questions/issues with this PR. 

Thanks!
Lauren

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bioconductor-banocc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - 'r-coda >=0.18.1'
+    - r-mvtnorm
+    - 'r-rstan >=2.10.1'
+    - r-stringr
+test:
+  commands:
+    - '$R -e "library(''{{ name }}'')"'
+about:
+  home: 'http://bioconductor.org/packages/{{ bioc }}/bioc/html/{{ name }}.html'
+  license: 'MIT + file LICENSE'
```
Comment (bgruening) [inline recipes/bioconductor-banocc/meta.yaml]: Can you please change the license to MIT?

**Comment by ljmciver** [comment]
Hi @bgruening - Thanks for the quick review! I just updated the license. Please let me know if there is anything else you would like changed.

Thanks,
Lauren

**Comment by bgruening** [comment]
Thanks @ljmciver!

**Comment by ljmciver** [comment]
Hi @bgruening - I just updated the kneaddata recipe. I would like to be added to the bioconda github team so I can continue to work on pull requests to update the existing bioBakery recipes. If I was on the team I can review / approve my own pull requests which will make it easier for me to help by 1) updating the existing bioBakery recipes 2) adding additional bioBakery recipes and 3) helping with maintenance / issues and updates of these recipes in the future. 

Sorry for the direct ping. I have tried other methods but I can't seem to reach a core member of the bioconda team.

Thank you,
Lauren

**Comment by bgruening** [comment]
Hi @ljmciver, I sent you an invite. But please do not merge your own PRs :) That is not the idea, we will try to review them asap.

Thanks for offering your help!

**Comment by ljmciver** [comment]
Hi @bgruening - Thank you! Oh, yes sorry. I will do that. Thank you for the heads up. Would it be okay if in the future I merge bioconda bot generated PRs for the biobakery recipes (those PRs that are automatically generated when the packages are updated in PyPi)?

Thanks again!
Lauren

**Comment by bgruening** [comment]
Yes of course! :)
But do not merge, use the bot to merge or assign the auto-merge tag.

**Comment by ljmciver** [comment]
@bgruening Got it! Thank you.

**Comment by ljmciver** [comment]
@bgruening Sorry for the ping. I tried to accept my invite but it has expired. Will you issue another one?

**Pull request merged**